Edit chapter 14

edilmedeiros opened the pull request:
I'll keep this as a draft for now, see #36.

edilmedeiros committed:
Break long paragraph lines

edilmedeiros committed:
Minor text edit

sb1752 commented:
Yeah if you want to go ahead and update the output in this PR based on that issue that's fine by me

edilmedeiros committed:
Fix console output messages

edilmedeiros commented:
Right, fixed the console messages.
Note that I also added a `-testnet` flag to the `bitcoin-cli` tool since this is a testnet transaction.
Finally, dropped any references to `script_sig` variables (see #39).

sb1752 commented on 14_json_serialization.md:
````diff
@@ -1,8 +1,9 @@
+For example:
 ```shell
-$ bitcoin-cli decoderawtransaction 010000000242d5c1d6f7308bbe95c0f6e1301dd73a8da77d2155b0773bc297ac47f9cd7380010000006a4730440220771361aae55e84496b9e7b06e0a53dd122a1425f85840af7a52b20fa329816070220221dd92132e82ef9c133cb1a106b64893892a11acf2cfa1adb7698dcdc02f01b0121030077be25dc482e7f4abad60115416881fe4ef98af33c924cd8b20ca4e57e8bd5feffffff75c87cc5f3150eefc1c04c0246e7e0b370e64b17d6226c44b333a6f4ca14 ...
+$ bitcoin-cli -testnet decoderawtransaction 010000000242d5c1d6f7308bbe95c0f6e1301dd73a8da77d2155b0773bc297ac47f9cd7380010000006a4730440220771361aae55e84496b9e7b06e0a53dd122a1425f85840af7a52b20fa329816070220221dd92132e82ef9c133cb1a106b64893892a11acf2cfa1adb7698dcdc02f01b0121030077be25dc482e7f4abad60115416881fe4ef98af33c924cd8b20ca4e57e8bd5feffffff75c87cc5f3150eefc1c04c0246e7e0b370e64b17d6226c44b33 ...
````
So since this is just decoding a raw transaction, it shouldn't really matter which network its running. And this will also fail if `bitcoind` is already running in some other mode such as `regtest`. For example locally I get 

```console
************************
EXCEPTION: St13runtime_error       
Invalid combination of -regtest, -signet, -testnet and -chain. Can use at most one.       
bitcoin in AppInitRPC()       
```

Might be best to leave it out unless I'm misunderstanding something.

edilmedeiros replied:
You are right. My node was running in testnet when I tried the command and it fooled me.
Fixed on the repo update.

edilmedeiros commented:
Removed the commit with the `-testnet` flag.

sb1752 commented:
Ack d5cccf76a351fc60da2ee0913b210138529dd8e0

Perfect thank you
